terraform/module.go

```diff
@@ -107,6 +118,7 @@ func WithOptionalInputs(oi []*Input) SectionFn {
 type module struct {
 	Header       string
 	Inputs       []*Input
+	Modulecalls  []*Modulecall
```

Comment:
I think it should be better to use full camel case `ModuleCalls` rather than `Modulecalls`.